interfaces/udev/spec.go

```diff
@@ -73,10 +81,12 @@ func udevTag(securityTag string) string {
 // TagDevice adds an app/hook specific udev tag to devices described by the
 // snippet and adds an app/hook-specific RUN rule for hotplugging.
 func (spec *Specification) TagDevice(snippet string) {
```

Comment:
Nitpick, `if spec.controlsDeviceCgroup { return }`

Reply:
gofmt says I should use ```go
func (spec *Specification) TagDevice(snippet string) {
	if spec.controlsDeviceCgroup {
		return
	}
	for _, securityTag := range spec.securityTags {
	...
```

is this okay or do you want it on one line?